blob/drivertest/drivertest.go

```diff
@@ -561,62 +564,62 @@ func testListDelimiters(t *testing.T, newHarness HarnessMaker) {
 		},
 		// TODO(#905): Backslashes cause problems for Azure; disable for now.
 		/*
-		{
-			name:  "backslash",
-			delim: "\\",
-			wantFlat: []listResult{
-				listResult{Key: keyPrefix + "\\dir1\\a.txt"},
-				listResult{Key: keyPrefix + "\\dir1\\b.txt"},
-				listResult{Key: keyPrefix + "\\dir1\\subdir\\c.txt"},
-				listResult{Key: keyPrefix + "\\dir1\\subdir\\d.txt"},
-				listResult{Key: keyPrefix + "\\dir2\\e.txt"},
-				listResult{Key: keyPrefix + "\\f.txt"},
-			},
-			wantRecursive: []listResult{
-				listResult{
-					Key:   keyPrefix + "\\dir1\\",
-					IsDir: true,
-					Sub: []listResult{
-						listResult{Key: keyPrefix + "\\dir1\\a.txt"},
-						listResult{Key: keyPrefix + "\\dir1\\b.txt"},
-						listResult{
-							Key:   keyPrefix + "\\dir1\\subdir\\",
-							IsDir: true,
-							Sub: []listResult{
-								listResult{Key: keyPrefix + "\\dir1\\subdir\\c.txt"},
-								listResult{Key: keyPrefix + "\\dir1\\subdir\\d.txt"},
+			{
```

Comment:
Note for reviewers: I must have not run `gofmt` when I commented this out before. You can set github to `No Whitespace` for diffs to hide it.